### PR TITLE
chore: upgrade deps

### PR DESCRIPTION
**Pull request by opqdonut**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -18,7 +18,12 @@ We use [Break Versioning][breakver]. The version numbers follow a `<major>.<mino
 * Updated dependencies:
 
 ```clojure
-[metosin/malli "0.10.4"] is available but we use "0.10.1"
+[metosin/ring-swagger-ui "4.18.1"] is available but we use "4.15.5"
+[metosin/malli "0.11.0"] is available but we use "0.10.4"
```
Comment (ikitommi) [inline CHANGELOG.md:22]: was "0.10.1".

File: CHANGELOG.md [hunk 2]
```diff
@@ -18,7 +18,12 @@ We use [Break Versioning][breakver]. The version numbers follow a `<major>.<mino
...
+[ring/ring-core "1.10.0"] is available but we use "1.9.6"
+[manifold "0.4.0"] is available but we use "0.3.0"
+[ring "1.10.0"] is available but we use "1.9.6"
+[nubank/matcher-combinators "3.8.5"] is available but we use "3.8.4"
```
Comment (ikitommi) [inline CHANGELOG.md:26]: this is a test dependency, should not be tracked.

File: CHANGELOG.md [hunk 3]
```diff
@@ -18,7 +18,12 @@ We use [Break Versioning][breakver]. The version numbers follow a `<major>.<mino
 * Updated dependencies:
 
 ```clojure
-[metosin/malli "0.10.4"] is available but we use "0.10.1"
+[metosin/ring-swagger-ui "4.18.1"] is available but we use "4.15.5"
+[metosin/malli "0.11.0"] is available but we use "0.10.4"
+[ring/ring-core "1.10.0"] is available but we use "1.9.6"
+[manifold "0.4.0"] is available but we use "0.3.0"
```
Comment (ikitommi) [inline CHANGELOG.md:24]: maybe test deps too?

File: CHANGELOG.md [hunk 4]
```diff
@@ -18,7 +18,12 @@ We use [Break Versioning][breakver]. The version numbers follow a `<major>.<mino
...
+[ring "1.10.0"] is available but we use "1.9.6"
```
Comment (ikitommi) [inline CHANGELOG.md:25]: test

File: CHANGELOG.md (hunk 2 above)
Comment (opqdonut) [reply CHANGELOG.md:26]: I'll remove it

File: CHANGELOG.md (hunk 1 above)
Comment (opqdonut) [reply CHANGELOG.md:22]: fixed

File: CHANGELOG.md (hunk 3 above)
Comment (opqdonut) [reply CHANGELOG.md:24]: yep

File: CHANGELOG.md (hunk 4 above)
Comment (opqdonut) [reply CHANGELOG.md:25]: yep

**Comment by opqdonut** [comment]
@ikitommi good to go now?

**Comment by ikitommi** [comment]
:+1:

**Pull request merged**